install/kubernetes/helm/istio/charts/kiali/templates/deployment.yaml

```diff
@@ -40,6 +40,16 @@ spec:
         - "/kiali-configuration/config.yaml"
         - "-v"
         - "4"
+        ports:
+        - containerPort: 20001
+        readinessProbe:
+          httpGet:
+            path: /kiali/console
```

Comment:
this context root of "/kiali" is customizable via helm so must be parameterized as you are not guaranteed "/kiali" is a valid path. See https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/kiali/values.yaml#L8

Reply:
I am deploying **Istio** in Azure AKS 1.12.5
I could be because of Dockerfile specs ( if it contains expose ports sections)! but as long as it's not explicitly defined in Kubernetes deployments! may cause unexpected behavior.
in my **case kubectl** port-forward did not respond properly so I traced deployed YAML files for root causing which in turn solved to adding **ports** specs into that. 
I mean in general! it is better to have it there! I believe...

Reply:
also about readiness/liveness probe path! for sure... I will look into other sub-charts and will update it simular to the rest of sub-charts! customizable via value files.